docs/Reference/RPC/commandsRPC.md

```diff
@@ -82,16 +84,30 @@ Deactivate the device:
     .\rpc deactivate -u wss://server/activate
     ```
 
+#### Deactivate the device locally (without RPS):
+
+=== "Linux"
+    ``` bash
+    sudo ./rpc deactivate -local
+    ```
+=== "Windows"
+    ```
+    .\rpc deactivate -local
+    ```
+
 | OPTION             | DESCRIPTION                                                                             |
 |--------------------|-----------------------------------------------------------------------------------------|
-| -f                 | Force deactivate even if device is not registered with a server                         |
+| -f                 | Force deactivate even if device is not registered with the RPS server                   |
 | -json              | JSON output                                                                             |
-| -l  string         | Log level (panic,fatal,error,warn,info,debug,trace) (default "info")                    |
+| -l string          | Log level (panic,fatal,error,warn,info,debug,trace) (default "info")                    |
 | -lmsaddress string | LMS address (default "localhost"). Can be used to change location of LMS for debugging. |
 | -lmsport string    | LMS port (default "16992")                                                              |
+| -local             | Deactivate AMT locally without the use of RPS. CCM Mode deactivation only.              |
```

Comment:
This is what we are using local variable description. "Execute command to AMT directly without cloud interaction". Would you like to sync it?